src/app/components/admin/page-admin-articles/page-admin-articles.component.html

```diff
@@ -0,0 +1,38 @@
+<ksi-admin-sidebar></ksi-admin-sidebar>
+<div class="content-container">
+    <div class="content">
+
+        <h2>Správa článků</h2>
+        <a class="btn btn-ksi"
+            [routerLink]="['/', routes.routes.admin._, routes.routes.admin.articles._, routes.routes.admin.articles.edit, '0']">
+            {{ icon.ADD }} {{ 'admin.articles.new' | translate }}</a>
+
+        <table class="table table-striped table-hover table-responsive-md threads table-dark">
```

Comment:
Místo table-dark používej prosím `ksiTheme`, které automaticky nastaví světlý/tmavý mód

```suggestion
        <table class="table table-striped table-hover table-responsive-md threads" ksiTheme>
```